db/models/cache.go

```diff
@@ -0,0 +1,16 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+type Cache struct {
+	gorm.Model
+	Key  string `gorm:"unique_index, type:varchar(64)"`
+	Data []byte
+}
+
+type CachingInterval struct {
+	gorm.Model
+	Timestamp int64
```

Comment:
can we use `time` format instead? as it's preferred way for postgress

Reply:
sure